analyzer/consensus/consensus.go

```diff
@@ -453,14 +484,14 @@ func (m *Main) prepareRegistryData(ctx context.Context, height int64, batch *sto
 		return err
 	}
 
-	for _, f := range []func(*storage.QueryBatch, *storage.RegistryData) error{
+	for _, f := range []func(*storage.QueryBatch, *storage.RegistryData, map[string]int) error{
 		m.queueRuntimeRegistrations,
 		m.queueRuntimeStatusUpdates,
 		m.queueEntityEvents,
 		m.queueNodeEvents,
 		m.queueRegistryEvents,
```

Comment:
We don't need to feed `txHashIndexLookup` to all of these, right? It's misleading for them to consume the map but then not use it. Options:
- Have two for loops, one for `[]func(*storage.QueryBatch, *storage.RegistryData, map[string]int) error` and one for `[]func(*storage.QueryBatch, *storage.RegistryData)`
- If not that, let's at least prepend the arg name of `txHashIndexLoop` with `_` as a soft indication that it's unused. For example, `func (m *Main) queueRuntimeRegistrations(batch *storage.QueryBatch, data *storage.RegistryData, _txHashIndexLookup map[string]int)`

Reply:
Using two separate loops of events processing to bypass this problem.